### DISCUSSION TITLE
115-124 year old men in Slovakia?

### DISCUSSION THREAD

**Discussion by freedomlives** [Institut-Zdravotnych-Analyz/covid19-data#50] [General]
Looking at the Positive Tests - Age Group - District data, I found one man in each of the 120, 115, and 110 age groups. Which is kind of world record setting if true. Any way to tell if this was just mistakes in data entry? (There is one 995 entry, which I presume was a DE mistake, and not that Methusalah lives here)
![covid_tests](https://user-images.githubusercontent.com/16663872/107237703-4f33dd80-6a27-11eb-8541-8ac97c745e46.png)

**Comment by lukasito** [comment]
This definitely needs an answer. At least, if this is a mistake, if it was fixed or not.

**Comment by freedomlives** [comment]
Perhaps it was partly a data error. Now there is just one 115-119 year old, and two 110-114 year olds. Which is still a super impressive age.
